src/test/rgw/test_rgw_crypto.cc

```diff
@@ -24,8 +24,8 @@
 using namespace std;
 
 
-std::unique_ptr<BlockCrypt> AES_256_CBC_create(CephContext* cct, const uint8_t* key, size_t len);
-
+std::unique_ptr<BlockCrypt> AES_256_CBC_create(const DoutPrefixProvider *dpp, CephContext* cct, const uint8_t* key, size_t len);
+const DoutPrefixProvider *no_dpp(g_ceph_context, dout_subsys);
```

Comment:
This should be declared as a `NoDoutPrefix`. See `src/common/dout.h` on line 56 for the declaration of that class.